packages/dashboard-frontend/src/containers/Loader/Factory/__tests__/getGitRemotes.spec.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2018-2022 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { getGitRemotes, GitRemote } from '../getGitRemotes';
+
+describe('getGitRemotes()', () => {
```

Comment:
If the code looks ok, I plan to add/fix more tests

Reply:
I added feedback for 'sanitizeValue' https://github.com/eclipse-che/che-dashboard/pull/691/files#r1049678740